### ISSUE TITLE
Jetpack Monitor: settings aren't saved in Calypso

### ISSUE THREAD

**Issue by jeherve** [Automattic/wp-calypso#7243]
#### Steps to reproduce
1. You'll need a Jetpack site connected to WordPress.com, with the Monitor module active.
2. Starting at URL: https://wordpress.com/settings/security/yoursite.com
3. Scroll down to the Monitor section.
4. Uncheck email notifications.
5. Click on "Save Settings".
6. You'll receive a message that your settings have been saved.
7. Refresh the page.
8. The checkbox is active again.

![screen shot 2016-08-03 at 1 20 33 pm](https://cloud.githubusercontent.com/assets/426388/17363834/14d49af4-597d-11e6-8fc7-8f6901d8241b.png)

**Comment by deltaWhiskey** [comment]
Same experience here.  On step 5, the browser is reaching out to the server with POST parameters:
```
fetchingSettings:false
enabled:true
emailNotifications:false
```

But the server's response is not echoing back "emailNotifications" in the "updated" array.  HTTP response body for me was:

```
{
    "code": 200,
    "headers": [
        {
            "name": "Content-Type",
            "value": "application/json"
        }
    ],
    "body": {
        "updated": []
    }
}
```

**Comment by lancewillett** [comment]
Bug scrub note: I'm able to repeat this myself. Unchecking the box, save looks successful — but on reload the page shows a checked box again.

**Comment by lancewillett** [comment]
> But the server's response is not echoing back "emailNotifications" in the "updated" array. HTTP response body for me was:

API side issue maybe?

**Comment by lancewillett** [comment]
Jetpack plugin settings save successfully from `wp-admin/admin.php?page=jetpack#/security` — so it could be a bug in Calypso rather than an API bug, assuming Jetpack admin UI also calls the API endpoint, and not a Jetpack Monitor method directly.

**Comment by deltaWhiskey** [comment]
When I peeked at this earlier, it looked like the client side was working OK, but that the server had no routine to handle an emailNotifications update (at least, not in the server-side function being used by this form.) This looked like it could be fixed server-side by adding a handler for an emailNotifications update.

I believe the plugin setting page uses a different WordPress.com endpoint than Calypso.

**Comment by gravityrail** [comment]
I'm looking into this with @deltaWhiskey

**Comment by gravityrail** [comment]
@ebinnion has a fix for this in #9558
